sdk/core/core/inc/az_span.h

```diff
@@ -188,6 +188,17 @@ AZ_NODISCARD az_span az_span_from_str(char* str);
  */
 AZ_NODISCARD az_span az_span_slice(az_span span, int32_t low_index, int32_t high_index);
 
+/**
+ * @brief az_span_slice_start returns a new az_span which is a sub-span of the specified span.
+ *
+ * @param[in] span The original az_span
+ * @param[in] low_index An index into the original az_span indicating where the returned az_span
+ * will start.
+ * @return An az_span (view) into a portion (from low_index to the capacity) of the original
+ * az_span.
+ */
+AZ_NODISCARD az_span az_span_slice_start(az_span span, int32_t low_index);
```

Comment:
I don't know..  I see this name a little weird.
So, @JeffreyRichter ,  for Json, we changed from `az_json_object` to `az_json_object_start` to explicitly mention it's not an object but only the start.
Now, here we have this function `az_span_slice_start` that represents getting an az_span and not only the start of it...

This makes me thing about it...  I would probably prefer `az_span_from_start`, (or you know, the practical and always hared `az_span_drop`) :)

Reply:
I am still evaluating whether we want to add this API or not. I do like your suggestions of `az_span_slice_from_start` OR `az_span_slice_to_end` (suggested by Jeffrey), so if we were going to add this, I will go with one of those two suggestions instead of what I have right now :)